bindings/protocols/bacnet/index.html

```diff
@@ -1032,8 +1032,27 @@ <h3>Mappings related to events</h3>
                 <ol>
                     <li>Each Notification Class is mapped to an EventAffordance. This maps directly to the subscription granularity.</li>
                     <li>Event Enrollment objects used for algorithmic reporting are not explicitly mapped; they are not relevant to the WoT consumer, per se.</li>
-                    <li>Special event-related actions (getting current events and acknowledging events) are mapped to ActionAffordances.</li>
-
+                    <li>
+                        Special event-related actions (getting current events and acknowledging events) are mapped to ActionAffordances.
+                        <div class="note">
+                            <p>The WoT consumer does not need to know the details of BACnet alarming, it can simply interact with the InteractionAffordances of the Thing,
+                            as per their data schemata. However the consumer still needs to have an implicit understanding about the relations
+                            between the InteractionAffordances:</p>

```

Comment:
```suggestion
                            <p>The WoT consumer does not need to know the details of BACnet alarming; it can simply interact with the InteractionAffordances of the Thing,
                            according to their data schemata. However, the consumer does need to have an understanding of the relations
                            between the InteractionAffordances:</p>

```